consensus/reactor.go

```diff
@@ -794,6 +793,38 @@ OUTER_LOOP:
 	}
 }
 
+func (conR *ConsensusReactor) peerStatsRoutine() {
+	for {
+		// Manage disconnects from self or peer.
+		if !conR.IsRunning() {
+			conR.Logger.Info("Stopping peerStatsRoutine")
```

Comment:
What purpose does this line serve? If that is to inform developer (validators shouldn't care about it) about a successful exit, then I'd prefer to have leaktest to make sure go-routine exits instead of relying on human eye post-failure.